common-api/build.gradle

```diff
@@ -0,0 +1,71 @@
+buildscript {    // Configuration for building
+  repositories {
+    jcenter()    // Bintray's repository - a fast Maven Central mirror & more
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
+  dependencies {
+  }
+}
+
+apply plugin: 'java'
+apply plugin: 'eclipse'
+apply plugin: 'idea'
+
+
+repositories {   // repositories for Jar's you access in your code
+  jcenter()
+}
+
+dependencies {
+
+  // To show the dependency tree, try: ./project.rb gradle dependencies --configuration compile
+
+  compile 'joda-time:joda-time:+'
+  compile 'javax.inject:javax.inject:1'
+  compile 'com.google.guava:guava:22.0'
+  compile 'org.hibernate:hibernate-ehcache:5.0.12.Final'
+  compile 'org.apache.commons:commons-lang3:3.0'
+  compile 'org.springframework.data:spring-data-jpa:1.11.3.RELEASE'
```

Comment:
Does this order have meaning or would it be worth alphabetizing so we don't have to think about it?